sandbox/proxy/controller.go

```diff
@@ -43,6 +44,8 @@ func (s *remoteSandboxController) Create(ctx context.Context, sandboxID string,
 	for _, opt := range opts {
 		opt(&options)
 	}
+	// Save NetNSPath in ctx metadata
+	ctx = namespaces.WithCustomKeyValue(ctx, namespaces.NetNSPathKey, options.NetNSPath)
```

Comment:
I don't think we should add the helper in the namespaces. We should just implement the func in this package.

Reply:
Updated.